<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(20, 30, 50);
color: white;
</string>
  </property>
  <widget class="QLabel" name="com_port_label">
   <property name="geometry">
    <rect>
     <x>114</x>
     <y>80</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>COM Port</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="com_port_text">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>80</y>
     <width>50</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="capture_device_label">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>120</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Capture Device</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="capture_device_text">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>120</y>
     <width>50</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="control_method_label">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>160</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Control Method</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="ok_button">
   <property name="geometry">
    <rect>
     <x>162</x>
     <y>240</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QComboBox" name="control_method_box">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>160</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
